### ISSUE TITLE
Use "mocks" instead of "capable"

### ISSUE THREAD

**Issue by MichaelRFairhurst** [MichaelRFairhurst/wake-compiler#164]
Just a random thought. Everyone understands a/an in inheritance, but capably throws people off. Why not call it what it is? I mean usually mocks are reserved for testing but given wake is built for testing I see value in showing people the testy background.

    every FilePrinter (mocks Printer):

Deciding if I like this.

**Comment by RenanStigliani** [comment]
I think interchangeability of concepts between languages would be good, that way, my suggestion is `implements`:

```
every FilePrinter (implements Printer):
```

**Comment by MichaelRFairhurst** [comment]
Yeah, I think you're right. No need to reinvent this particular term, I'm doing that enough already with Text/String, and Num/Float. This term is more likely to simply alienate people.

Removing RFC and moving into V1 changes
